olp-cpp-sdk-core/include/olp/core/thread/Continuation.inl

```diff
@@ -29,29 +29,48 @@ class TaskScheduler;
 
 template <typename ResultType>
 void Continuation<ResultType>::Run() {
+  // Run should not start an execution if the `Finally` method is not called
+  // for the task continuation.
```

Comment:
We usually use Doxygen multiline comment style only to describe an APIs.
This comment describes not an API, but internal implementation details so that `//` fits much better here.